docs/themes/code_mode.rst

```diff
@@ -1,6 +1,52 @@
 .. vale off
 
-Code Mode
+Code mode
 ##########
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Code Mode is an option available in the Email and Landing Page edit Form. It allows you to create/insert/edit your content in HTML code. It's helpful for situations where you don't want to use a Mautic Theme and you want to use an HTML Theme copied from a ``3rd`` party Theme builder or if you enjoy editing HTML code so much.
+
+With Mautic 2.3.0, Code Mode replaced the full-page Froala (WYSIWYG) editor that sometimes altered the HTML code. Code Mode won't modify any of the code you paste in. The other option to edit a page/email content is to use the Builder. It uses Froala editor only to edit the text/image content, not the full page, so that it won't modify the page layout.
+
+Select the Code mode
+********************
+
+After creating or editing the page/email, you can select the Code Mode from the Theme selector. To open the Code Mode Builder, click the Builder button after selecting the Code Mode Theme option.
+
+Limitations
+===========
+
+If you use a Mautic Theme to create the page/email and you want to edit the HTML code of it in the Code Mode Builder, you can do so, but you can't switch back to the Theme again. Code Mode Builder requires editing of all content. Selecting a Theme refreshes the content to the default Theme HTML, so you'll lose your modifications.
```

Comment:
```suggestion
If you use a Mautic Theme to create the Landing Page/Email and you want to edit the HTML code of it in the code mode Builder, you can do so, but you can't switch back to the Theme again. The code mode Builder requires editing of all content. Selecting a Theme replaces the content to the default Theme HTML, so you'll lose your modifications.  Instead, to make small code changes to the page once a theme is selected, it is recommended to use the code mode built into the GrapesJS builder.
```

Recommend adding a screenshot here of the GrapesJS code mode, and linking to the docs on the GrapesJS builder.

Reply:
<img width="1680" alt="Screenshot 2023-02-20 at 10 40 18 PM" src="https://user-images.githubusercontent.com/46398983/220202223-744548c8-d1fd-442a-a73a-44bb8d7de4cd.png">
Do you mean this? @RCheesley